lib/serve/index.ts

```diff
@@ -91,6 +91,7 @@ export class LisaServeApplicationStack extends Stack {
                 excludePunctuation: true,
                 passwordLength: 16
             },
+            removalPolicy: RemovalPolicy.RETAIN_ON_UPDATE_OR_DELETE
```

Comment:
we're coming up with a new key every deployment so i'd expect that we're not ever "updating" it (secret name changes every time, so that's a [replacement](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-secretsmanager-secret.html#cfn-secretsmanager-secret-name) each time).

I think we can keep this for now, but for future iterations where we want the key to continuously update (if it's not actively doing that already), it's not super clear to me if we're going to have multiple key *names* to track  or if we will be updating the version/secret within a specific secret name.

Not really anything to action on right now, but something we should add for a backlog item to track to ensure that the key is working the way we expect it to.

Reply:
This was leftover from testing having the secret be named the same every time. It was an attempt to workaround the issue where the Authorizer environment variables weren't staying up-to-date with the latest secret name. With the addition of the dependency though I believe this is solved now though and this can be removed.